tests/sca_package/test_scanner.py

```diff
@@ -1,137 +1,102 @@
-import json
 from pathlib import Path
-
-import pytest
-from mock import AsyncMock, MagicMock
-from pytest_mock import MockerFixture
+import responses
 
 from checkov.sca_package.scanner import Scanner
 
+EXAMPLES_DIR = Path(__file__).parent / "examples"
 
-def test_setup_twistcli_exists(mocker: MockerFixture, tmp_path: Path):
-    # given
-    scanner = Scanner()
 
-    integration_mock = MagicMock()
-    mocker.patch(
-        "checkov.common.bridgecrew.vulnerability_scanning.integrations.package_scanning.package_scanning_integration.download_twistcli",
-        side_effect=integration_mock,
-    )
+def assert_equal_dicts(a, b):
```

Comment:
if we use a={'a': 1}, b= {'a': 2} we got False-Positive
I guess that you have any assumption regarding the dicts 'a' and 'b'. please write them in the documentation (or as a comment) of the function

Reply:
Need to fix this still @Eliran-Turgeman